contents/docs/deployment/deploy-digital-ocean.md

```diff
@@ -9,12 +9,66 @@ showTitle: true
 [Digital Ocean](https://digitalocean.com) is one of the most well-established Cloud Providers. Compared to AWS, where the amount of options and configuration can be overwhelming, Digital Ocean is generally simpler to use and faster to get running. 
 <br />
 
-### Deploying to Digital Ocean
+## Deploying Posthog backed by Clickhouse
```

Comment:
s/Clickhouse/ClickHouse/g

Reply:
This also seems like it should have the new one click deploy in there. I know this diff is 14 days old so that's probably the reason why it's stale :P